pom.xml

```diff
@@ -45,8 +45,19 @@
 		<pluginManagement>
 			<plugins>
 				<plugin>
+          <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
Should we switch to 16?

Reply:
Good idea! JDK 16 has been released today